subgraph/schema.graphql

```diff
@@ -142,16 +146,18 @@ type Court @entity {
 
 type Dispute @entity {
   id: ID!
+  disputeID: BigInt!
   court: Court!
   arbitrated: Arbitrable!
   period: Period!
   ruled: Boolean!
   currentRuling: BigInt!
   tied: Boolean!
   overridden: Boolean!
-  lastPeriodChange: BigInt!
-  lastPeriodChangeBlockNumber: BigInt!
   periodDeadline: BigInt!
+  periodNotificationIndex: BigInt!
+  lastPeriodChangeTs: BigInt!
```

Comment:
Could we go without the "Ts" suffix on `lastPeriodChange` ? Mostly to keep the same name as the one in the contract.

Reply:
done